src/app/static/src/app/components/outputTable/OutputTable.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+    <div class="row">
+        <div class="col-md-3">
+            <filters></filters>
+        </div>
+        <div id="chart" class="col-md-9">
+            <table-comp></table-comp>
+        </div>
+    </div>
+</template>
+
+<script setup lang="ts">
+    import Filters from "./Filters.vue"
+    import TableComp from "./Table.vue"
```

Comment:
Not a huge fan of naming the imports differently from the component file, why do this? Maybe there's not a huge benefit in this container component anyway, it's probably not going to do anything other than literally contain the two inner components, so maybe those could just sit directly in the tab, and then `Table` could become `OutputTable`?

Reply:
so I noticed the `ModelOutput.vue` file is getting a little big, I think it would be nice to eventually split it up a bit so it can just be concerned with few controls like show this plot if this tab is selected. I was thinking this component could be in charge of filtering the data based on filter selections!

I dont like naming the files differently, basically the html tag for `<table>` is overwriting the vue one, is there a simple way to fix this or do you think it is best to just name it something different? do you think maybe `TableData` is a better name or `TableDisplay` something like that?

Reply:
gone with `TableData` for now